code/modules/power/supermatter/supermatter.dm

```diff
@@ -476,6 +476,8 @@
 /obj/machinery/power/supermatter_shard/attack_paw(mob/user)
 	return attack_hand(user)
 
+/obj/machinery/power/supermatter_shard/attack_animal(mob/living/simple_animal/user)
+	return attack_hand(user)
```

Comment:
As a bonus, it prevents issues where attack_hand could be overridden and have completely unintended behavior, whereas if die_horribly was overridden it would be completely intentional